src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -115,7 +119,38 @@ registerAction2(class extends Action2 {
 		if (!instance) {
 			return;
 		}
-		AccessibleBufferContribution.get(instance)?.show();
+		await AccessibleBufferContribution.get(instance)?.show();
+	}
+});
+
+registerAction2(class extends Action2 {
+	constructor() {
+		super({
+			id: TerminalCommandId.GoToCommand,
+			title: { value: localize('workbench.action.terminal.goToCommand', 'Go to Command'), original: 'Go to Command' },
```

Comment:
This feels a little too close to scroll to previous/next command but they do pretty different things. How about something closer to "Navigate Accessible Buffer"?